test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -201,3 +201,24 @@ Describe "Validate about_help.txt under culture specific folder works" -Tags @('
         $help | Should BeExactly "Hello"
     }
 }
+
+Describe "Get-Help should find help info within help files" -Tags @('CI', 'RequireAdminOnWindows') {
+    It "Get-Help should find help files under pshome" {
+        $helpFile = "about_testCase.help.txt"
+        if (!(Test-Path $PSHOME\en-US))
```

Comment:
Is this test case only meant to run on en-US? If it is, then you should skip it for non-English runs. Otherwise, please use (get-uiculture).Name.